src/components/LocationDetails.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function LocationDetails(props) {
+  const { city, country, errorMessage } = props;
+  return errorMessage ? (
+    <h1>{errorMessage}</h1>
+  ) : (
+    <h1 className="location-details">{`${city}, ${country}`}</h1>
```

Comment:
There is no class `location-details` defined anywhere in the project, and no style file associated with `LocationDetails`, so this classname is not adding any styling. Did you forget to add the styling? If not, this class could be removed and the return statement could be simplified to a one liner: 
```
return <h1>{errorMessage || `${city}, ${country}`}</h1>;
```